### PR TITLE
Automate GTM

### PR DESCRIPTION
**Pull request by adamread**
This adds a tool to automate setting up GTM to send CWV data to GA. The web frontend is super simple, and the JS focuses on using the GTM API to add the required bits and pieces.

### CODE REVIEW

**Review by notmariazoe — COMMENTED** [review]
Looks good to me- it would be great to have more information on what the two tags do.

File: gtm_automation/index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        color: green;
+      }
+      .error {
+        color: red;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Deploy the gPS Core Web Vitals GTM Tag</h1>
+    <p>Please fill out all of the fields before clicking the <strong>Deploy GTM Tag</strong> button. For more information, please see the <a href="docs/index.html">documentation</a>.</p>
```
[the remark this answers is not in the document]
Comment (adamread) [reply gtm_automation/index.html:46]: Yes. I need to update the main README and figure out how to join this to the work that's already been done.

File: gtm_automation/gtm_automation.js [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+ * Core Web Vitals in GA4 to Google Tag Manager.
+ */
+
+let tokenClient;
+let customEventTriggerId;
+
+/**
+ * Adds a message to the success-messages div on the page.
+ *
+ * @param {str} message The message to add to the page.
```
[the remark this answers is not in the document]
Comment (adamread) [reply gtm_automation/gtm_automation.js:27]: thanks

File: gtm_automation/gtm_automation.js [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+      .then(function () {
+        gapi.client.load('https://www.googleapis.com/discovery/v1/apis/tagmanager/v2/rest');
+      })
+      .then(deployTag());
+  });
+}
+
+/**
+ * Deploys the gPS Core Web Vitals Tag to the GTM workspace as defined on the
+ * page. If completed successfully, deployEventTrigger() is called.
```
[the remark this answers is not in the document]
Comment (adamread) [reply gtm_automation/gtm_automation.js:71]: Updated the docs to, hopefully, make this more clear.

File: gtm_automation/gtm_automation.js [hunk 4]
```diff
@@ -0,0 +1,412 @@
...
+          value: cwvTagValue,
+        },
+        {
+          'type': 'boolean',
+          key: 'supportDocumentWrite',
+          value: 'false',
+        },
+      ],
+      firingTriggerId: [
+        "2147479553"
```
[the remark this answers is not in the document]
Comment (adamread) [reply gtm_automation/gtm_automation.js:99]: Added a comment.

**Review by notmariazoe — APPROVED** [review]

**Pull request merged**